recipes/delete-me/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "giotto-learn" %}
+{% set version = "0.1a" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/matteocao/{{ name }}.git
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script:
+    - git submodule update --init --recursive
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - gcc  # [not win]
+    - vs2017  # [win]
+    - cmake
+    - boost >=1.56.*
```

Comment:
Please remove ``boost`` from here, it should be just in ``host?
why do you need ``git``?
Please remove the ``setuptools`` as well, it also should be just in ``host``